client/utils.py

```diff
@@ -148,6 +149,33 @@ def write_command(fdesc, command, destinations, source=0):
     time.sleep(0.3)
 
 
+def write_command_retry(fdesc, command, destinations, source=0):
+    """
+    Writes a command, retries as long as there is no answer and returns a dictionnary containing
+    a map of each board ID and its answer.
+    """
+    write_command(fdesc, command, destinations, source)
+    reader = CANDatagramReader(fdesc)
+    answers = dict()
+
+    while len(answers) < len(destinations):
```

Comment:
Well I thought the retry limit was the user hitting Ctrl-C ;). What do you think would be a reasonable behaviour ?